extensions/dicom-seg/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@ohif/extension-dicom-seg",
```

Comment:
@JamesAPetts, just want to be sure this is the name we want. This does add a sopClassHandler for "DICOM SEG" but it also contains UI and tool code that's tightly coupled to `cornerstone-tools`

Reply:
To be useable its going to be tied to cornerstone tools for the foresable future. If this pattern ever changes it can have a major version bump.

Reply:
@dannyrb pending? let me know if theres something to be done